scripts/scripts.js

```diff
@@ -354,6 +354,39 @@ function decorateH3(main) {
     });
   }
 }
+
+/**
+ * Checks for the same domain or not and if ends with pdf
+ */
+function isSameDomainOrPdf(url) {
+  const ancUrl = (url.indexOf('://') > 0 || url.indexOf('//') === 0)
+    ? new URL(url)
+    : new URL(window.location.origin + url);
+  return (
+    window.location.hostname === ancUrl.hostname
+    || ancUrl.pathname.toLowerCase().endsWith('.pdf')
+    || ancUrl.pathname.toLowerCase().endsWith('.hlx.live')
+    || ancUrl.pathname.toLowerCase().endsWith('.hlx.page')
+  );
+}
+
+function decorateLinks(main) {
+  // Get all anchor elements within the main container
+  const links = main.querySelectorAll('a');
+
+  // Loop through each anchor element and add a target based on the business condition
+  links.forEach((link) => {
+    const href = link.getAttribute('href');
```

Comment:
If you use `link.href` instead of `link.getAttribute('href')` you would not need this logic https://github.com/hlxsites/servicenow/pull/116/files#diff-fbceb58f32b99b8a5309251f8c49ac8eda82fba58b905c600c92bce1ed2a1c8cR362-R364 , because it will return a fully qualified url (including the domain for relative links)